jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,891 @@
+* documentation and minor improvements to 170 modules, including 312 new runnable examples
+* 280 new nimble packages
+
+Nim made its first entry in TIOBE index in 2017 at position 129,
+last year it entered the top-100, and in the last 2 months the top-50 ([https://forum.nim-lang.org/t/8297](https://forum.nim-lang.org/t/8297)).
+We hope this release will reinforce this trend, building on Nim's core strengths:
+a practical, compiled systems programming language; offering C-like performance and portability;
+Python-like syntax; Lisp-like flexibility; strong C, C++, JS, Python interop;
+and best-in class metaprogramming.
+
+Why use Nim?
+* one language to rule them all: from [shell scripting](https://nim-lang.org/docs/nims.html) to [web frontend and backend](https://github.com/nim-lang/nimforum), [scientific computing](https://github.com/SciNim), [machine learning](https://github.com/mratsim/Arraymancer), [blockchain client](https://github.com/status-im), [gamedev](https://github.com/ftsf/nico), [embedded](https://github.com/EmbeddedNim).
+* concise and readable like Python: `echo "hello world"` is a 1-liner (5 lines in C, C++, Go, Java).
+* small binaries: `echo "hello world"` generates a 73K binary (Go: 2MB, Rust: 377K, C++: 56K) [1].
+* fast compile times: a full compiler rebuild takes ~12s (Rust: 15min, gcc: 30min+, clang: 1hr+, Go: 90s) [2].
+* C-like performance: see [Web Frameworks Benchmark](https://web-frameworks-benchmark.netlify.app/result), [ray tracing](https://nim-lang.org/blog/2020/06/30/ray-tracing-in-nim.html), [primes](https://github.com/PlummersSoftwareLLC/Primes)
+* no need for makefiles, cmake, configure or other build scripts, thanks to CTFE and dependency tracking [3]
+* target any platform with a C compiler: [embedded systems](https://github.com/elcritch/nesper), [micro-controllers](https://forum.nim-lang.org/t/7731), [WASM](https://forum.nim-lang.org/t/4779), Nintendo Switch or [Game Boy Advance](https://forum.nim-lang.org/t/8375)
+* zero-overhead interop lets you reuse code in C, C++ (including templates, [C++ STL](https://clonkk.github.io/nim-cppstl/cppstl.html)), JS, Objective-C, Python (via [nimpy](https://github.com/yglukhov/nimpy))
+* built-in [documentation generator](https://nim-lang.github.io/Nim/system.html) that understands Nim code and runnable examples that stay in sync
+
+Last but not least, macros let you manipulate/generate code at compile time instead of relying on code generators, allowing for example to easily write DSLs or extend the language, all in user code:
+  - [karax](https://github.com/karaxnim/karax) for react+JSX-like Single Page Applications
+  - [cligen](https://github.com/c-blake/cligen) for API-inferred command line generator
+  - [std/wrapnils](https://nim-lang.github.io/Nim/wrapnils.html) JS-like optional chaining
+  - [std/strformat](https://nim-lang.github.io/Nim/strformat.html) Python-like f-strings
```

Comment:
This is too much. This post should focus on the news, the blurb above is enough to entice those that don't know what Nim is about it. I would suggest replacing all of this with a link to your front page:

> Want to learn more about Nim and why you should consider using it? Take a look at a comprehensive list of its features [here][(https://nim-lang.org)

Reply:
also it might be better to write a dedicated article for Goodboy (The Gameboy Advance game developed by @exelotl )

Reply:
I've shortened things a bit by deferring some items/links a footnote; we can improve/shorten things a bit further but this is an opportunity to advertise in particular with people less familiar with nim (eg too lazy to click on the website or who will be skimming through this on HN and have no idea about nim); ppl already familiar with nim will just skip this paragraph;

Overall, 18 lines out of 955 lines on "why use nim" is not an excessive amount IMO.

> also it will likely be better to write a dedicated article for Goodboy (The Gameboy Advance game developed by @exelotl)

a dedicated article on `Goodboy` would definitely be good to have (but it doesn't hurt to mention it here)

Reply:
> Overall, 18 lines out of 955 lines on "why use nim" is not an excessive amount IMO.

18 lines isn't excessive. But 955 sure is :D

I've proof-read the article and got bored reading it before I reached 50% of the content.